.github/workflows/linters.yml

```diff
@@ -0,0 +1,161 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: "Linters"
+
+on:
+  push:
+    branches:
+      - "main"
+      - "v2.**"
+  pull_request: null
+# Add [skip ci] to commit message to skip CI.
+
+permissions: {}
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
```

Comment:
I think the other workflows need this too.

Reply:
Done.